.gitattributes

```diff
@@ -0,0 +1,149 @@
+# Auto-detect text files
+* text=auto
+
+# Some source files are edited with LF endings by everyone
+.editorconfig eol=lf
+.eslintrc eol=lf
+.gitignore eol=lf
+.mention-bot eol=lf
+COMMIT_EDITMSG eol=lf
+Dockerfile eol=lf
+git-rebase-todo eol=lf
+LICENSE eol=lf
+makefile eol=lf
+*.c eol=lf
+*.cc eol=lf
+*.clj eol=lf
+*.coffee eol=lf
+*.cpp eol=lf
+*.cs eol=lf
+*.cshtml eol=lf
+*.css eol=lf
+*.desktop eol=lf
+*.diff eol=lf
+*.fs eol=lf
+*.go eol=lf
+*.groovy eol=lf
+*.handlebars eol=lf
+*.hbs eol=lf
+*.hlsl eol=lf
+*.html eol=lf
+*.ini eol=lf
+*.java eol=lf
+*.js eol=lf
+*.json eol=lf
+*.jsx eol=lf
+*.less eol=lf
+*.lua eol=lf
+*.m eol=lf
+*.md eol=lf
+*.opts eol=lf
+*.php eol=lf
+*.pl eol=lf
+*.plist eol=lf
+*.pug eol=lf
+*.py eol=lf
+*.r eol=lf
+*.rb eol=lf
+*.recipe eol=lf
+*.rs eol=lf
+*.scss eol=lf
+*.shader eol=lf
+*.sql eol=lf
+*.svg eol=lf
+*.swift eol=lf
+*.template eol=lf
+*.tmLanguage eol=lf
+*.tmLanguage.base eol=lf
+*.tmTheme eol=lf
+*.ts eol=lf
+*.ts.disabled eol=lf
+*.tsx eol=lf
+*.txt eol=lf
+*.vb eol=lf
+*.vscodeignore eol=lf
+*.xml eol=lf
+*.xpm eol=lf
+*.yaml eol=lf
+*.yml eol=lf
+
+# Some source files are edited with CRLF endings by everyone
+LICENSE.txt eol=crlf
+ThirdPartyNotices.txt eol=crlf
+*.isl eol=crlf
+*.iss eol=crlf
+*.jxs eol=crlf
+
+# Some scripts require specific endings, others use specific in vscode
+*.bat eol=lf
+*.cmd eol=crlf
+*.ps1 eol=lf
```

Comment:
:memo: Normally this would use `eol=crlf`, but the repository is not currently following this convention.